portal/views/healthcheck.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+from flask import Blueprint, current_app
+import redis
+from sqlalchemy import text
+
+from ..factories.celery import create_celery
+from ..database import db
+
+
+healthcheck_blueprint = Blueprint('healthcheck', __name__)
+
+
+@healthcheck_blueprint.route('/celery_beat_ping')
+def celery_beat_ping():
+    """Periodically called by a celery beat task
+
+    Updates the last time we recieved a call to this API.
+    This allows us to monitor whether celery beat tasks are running
+    """
+    rs = redis.StrictRedis.from_url(current_app.config['REDIS_URL'])
+    rs.setex(
+        'last_celery_beat_ping',
+        current_app.config['LAST_CELERY_BEAT_PING_EXPIRATION_TIME'],
```

Comment:
Can you use keyword arguments for these, so it's obvious what parameter is which?

Reply:
for reference:
```
setex(self, name, value, time) method of redis.client.Redis instance
    Set the value of key ``name`` to ``value`` that expires in ``time``
    seconds. ``time`` can be represented by an integer or a Python
    timedelta object.
```